troubleshooting/troubleshooting.md

```diff
@@ -302,7 +302,7 @@ Outdated OpenRuntime.efi, make sure BOOTx64.efi, OpenCore.efi and OpenRuntime ar
 * [`kextd stall[0]: AppleACPICPU`](../troubleshooting/troubleshooting.md#kextd-stall0-appleacpicpu)
 * [MediaKit reports not enough space](../troubleshooting/troubleshooting.md#mediakit-reports-not-enough-space)
 * [DiskUtility failing to erase](../troubleshooting/troubleshooting.md#diskutility-failing-to-erase)
-
+* [SATA Drives Not Shown in Disk Utility](../troubleshooting/troubleshooting.md#diskutility-not-showing-sata-drives)
```

Comment:
This doesn't match with the header, the hyperlink will fail in this case. Should be this:
```
../troubleshooting/troubleshooting.md#sata-drives-no-shown-in-disk-utility
```